appveyor.yml

```diff
@@ -20,16 +20,20 @@ install:
   - "curl -O https://www.sqlite.org/2016/sqlite-dll-win32-x86-3110000.zip"
   - "7z x sqlite-dll-win32-x86-3110000.zip"
   - "copy /y sqlite3.dll %PYTHON%\\DLLs\\"
-  
+
   - "%PYTHON%\\python.exe -m pip install -e pkg"
   - "%PYTHON%\\python.exe -m pip install -e master[tls,test]"
   - "%PYTHON%\\python.exe -m pip install -e worker"
+  - "%PYTHON%\\python.exe -m pip install coverage codecov"
   - "%PYTHON%\\python.exe -m pip list"
 
 build: false
 
 test_script:
-  - "%PYTHON%\\python.exe -m twisted.trial --reporter=text --rterrors buildbot.test buildbot_worker.test"
+  - "%PYTHON%\\Scripts\\coverage.exe run --rcfile=.coveragerc -m twisted.trial --reporter=text --rterrors buildbot.test buildbot_worker.test"
```

Comment:
AppVeyor reports:

    Couldn't read config file .coveragerc: File contains no section headers.
    file: .coveragerc, line: 1
    'common/coveragerc'

I think this is because `.coveragerc` is a symlink and it appears git or Windows isn't translating that properly. Maybe try using its destination, `common/coveragerc`?